packages/react-tinacms-github/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "react-tinacms-github",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```
as per:
https://github.com/tinacms/tinacms/pull/958#discussion_r401661039